src/logging.rs

```diff
@@ -56,7 +56,7 @@ macro_rules! infoheader {
 
 #[macro_export]
 macro_rules! infoentry {
-	($str:expr, $rhs:expr) => ($crate::infoentry!($str, "{}", $rhs));
+	($str:expr, $rhs:expr) => (crate::infoentry!($str, "{}", $rhs));
```

Comment:
This should stay `$crate`, right?

```suggestion
	($str:expr, $rhs:expr) => ($crate::infoentry!($str, "{}", $rhs));
```

Reply:
Or does this not work with `$crate`?